src/TrackerCouncil.Smz3.UI/Services/MainWindowService.cs

```diff
@@ -141,6 +139,7 @@ public async Task DownloadSpritesAsync()
             InitialJsonPath = RandomizerDirectories.SpriteInitialJsonFilePath,
             ValidPathCheck = p => Sprite.ValidDownloadExtensions.Contains(Path.GetExtension(p).ToLowerInvariant()),
             ConvertGitHubPathToLocalPath = p => p.Replace("Sprites/", ""),
+            DeleteExtraFiles = RandomizerDirectories.DeleteSprites
```

Comment:
We're not at risk of deleting anybody's custom sprites here, right? This is deleting the old directory only?

Reply:
Not anymore than before I don't think. I'm pretty certain installs on Windows actually delete the old sprites in the localappdata/programs/SMZ3CasRandomizer folder. User sprites go into the locallappdata/SmZ3CasRandomizer folder .